.ko.yaml

```diff
@@ -2,3 +2,6 @@ defaultBaseImage: registry.access.redhat.com/ubi8/ubi-minimal
 
 baseImageOverrides:
   github.com/shipwright-io/build/cmd/git: ghcr.io/shipwright-io/base-git:latest
+  ## TODO: create a base image for waiter to maintain "gnu/tar" installed.
+  # github.com/shipwright-io/build/cmd/waiter: ghcr.io/shipwright-io/base-waiter:latest
+  github.com/shipwright-io/build/cmd/waiter: docker.io/alpine:latest
```

Comment:
You may consider to quickly split out the build of the new base image out into a separate PR, so that you can reference it here.

Reply:
Thanks. I'll mark this as resolved.